app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/DownloadSingleButton.tsx

```diff
@@ -29,12 +29,27 @@ export const DownloadSingleButton = ({
 
   const handleDownload = async () => {
     try {
+      const extraTranslations = {
+        streetAddress: {
+          en: t("addressComponents.streetAddress"),
+          fr: t("addressComponents.streetAddress"),
+        },
+        city: { en: t("addressComponents.city"), fr: t("addressComponents.city") },
+        province: { en: t("addressComponents.province"), fr: t("addressComponents.province") },
+        postalCode: {
+          en: t("addressComponents.postalCode"),
+          fr: t("addressComponents.postalCode"),
+        },
+        country: { en: t("addressComponents.country"), fr: t("addressComponents.country") },
+      };
+
```

Comment:
Same comment as above, this stuff shouldn't be repeated across multiple unrelated components. i18n utils should be used.